core/arch/arm/tee/entry_std.c

```diff
@@ -221,6 +222,12 @@ static void entry_invoke_command(struct thread_smc_args *smc_args,
 	struct tee_dispatch_invoke_command_out out;
 	struct optee_msg_param *params = OPTEE_MSG_GET_PARAMS(arg);
 
+#ifdef CFG_TEE_BENCHMARK
+	if (params[TEE_BENCH_DEF_PARAM].u.tmem.buf_ptr)
+		tee_add_timestamp((void *)(uintptr_t)
+			params[TEE_BENCH_DEF_PARAM].u.tmem.buf_ptr,
+			TEE_BENCH_CORE);
+#endif
```

Comment:
> It seems that params[TEE_BENCH_DEF_PARAM].u.tmem.buf_ptr here is physical address, can we visit this address directly?

oh, thanks that you have noticed it! wrapped with phys_to_virt call
it's epic fail :)